virt/virt-2-4-release-notes.adoc

```diff
@@ -19,6 +19,10 @@ You can use {VirtProductName} with either the xref:../networking/ovn_kubernetes_
 == New and changed features
 //Placeholder for new content.
 
+//CNV-5459 - Installation of Openshift virtualization using the CLI with kustomize is now supported.
+
+* You can now xref:../virt/install/installing-virt-cli.adoc#installing-virt-cli[install {VirtProductName} by using the CLI] to apply manifests to your {product-title} cluster.
```

Comment:
It would be good to specify what kind of manifests (since most objects in ocp is created by applying a manifest via the CLI)
Maybe Subscription and Operator manifests?

Reply:
Leaving this as-is for now based on the Slack chat.